pkg/cfn/manager/api.go

```diff
@@ -54,6 +57,7 @@ func NewStackCollection(provider api.ClusterProvider, spec *api.ClusterConfig) *
 	}
 	logger.Debug("tags = %#v", tags)
 	return &StackCollection{
+		cfnSvcRoleArn: provider.CloudFormationRoleARN(),
```

Comment:
Seems redundant, as we already have `provider` and can call the `CloudFormationRoleARN` on it to obtain the role ARN.